template/README.md

```diff
@@ -340,6 +341,9 @@ terraform apply
 > :bulb: `default` is the name of running VM. You could also use VM `id`.
 To get vm `id` check `vagrant global-status`.
 
+### Module versioning
+Remember to update the changelog and label the PR as a release (major/minor/patch). The CHANGELOG.md should be in this [syntax](https://keepachangelog.com/en/1.0.0/).
```

Comment:
The template workflow does actually not have a "release" workflow, so there is no reason to add a major/minor/patch tag. 
```suggestion
### Changelog
The CHANGELOG.md should follow this [syntax](https://keepachangelog.com/en/1.0.0/).
```